_nodes/0023.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Opposite fibrations
```

Comment:
I think we need to call this something different (and also rename the macro, or perhaps not even have a macro).

Reply:
Yes, please suggest something!

Reply:
"Global opposite" or "Total opposite"?